runtime/runtime-params-estimator/src/action_costs.rs

```diff
@@ -60,10 +60,13 @@ struct ActionEstimation {
     outer_iters: usize,
     /// how many iterations to ignore for measurements
     warmup: usize,
+    /// By default, the average cost is reported. Set this to true to report the
+    /// 75th percentile instead. This only affects outer iterations.
+    report_percentile: bool,
```

Comment:
An idea (feel free to skip): We can use Rust enums instead of boolean here to make this self-descriptive and a bit more general (though that violates YAGNI a bit):
```rust
enum Statistic {
    Average,
    Percentile(f32),
}
```

Reply:
Yeah, could be reasoned either way but I now added the enum as I think it looks quite a bit nicer this way.